### PR TITLE
Add OpenSea as limited

### PR DESCRIPTION
**Pull request by WebSnke**

### CODE REVIEW

**Review by tupaschoal — COMMENTED** [review]
Can you elaborate on why it is limited or what is the limit on the `notes`?

**Comment by WebSnke** [comment]
> Excerpt from [OpenSea's Privacy Policy](https://opensea.io/privacy):
> 
> > **Data Access and Control.** You can view, access, edit, or delete your Personal Data for certain aspects of the Service via your Settings page. You may also have certain additional rights:
> > - If you wish to exercise your rights under the GDPR, CCPA, or other applicable data protection or privacy laws, please contact us by using the “Submit a request” link here or at the address provided in Section 13 below, specify your request, and reference the applicable law. We may ask you to verify your identity, or ask for more information about your request. We will consider and act upon any above request in accordance with applicable law. We will not discriminate against you for exercising any of these rights.

Yep, that's why it should be "limited". You seem to be able to delete your account only if you live in one of these Regions.

**Comment by tupaschoal** [comment]
Can you add a "notes" field with a summarised version of that, then, please?

**Comment by WebSnke** [comment]
> Can you add a "notes" field with a summarised version of that, then, please?

Ok, I just did.

File: _data/sites.json [hunk 1]
```diff
@@ -11320,6 +11320,17 @@
...
     },
 
+    {
+        "name": "OpenSea",
+        "url": "https://support.opensea.io/hc/requests/new",
+        "difficulty": "limited",
+        "notes": "Deleting your account is only possible if an applicable government policy gives you the right to delete your account.",
+        "domains": [
+            "opensea.io/",
+            "support.opensea.io/"
```
Comment (tupaschoal) [inline _data/sites.json:11330]:
```suggestion
            "opensea.io",
            "support.opensea.io"
```

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**